LIBVmotor0.11/LIBVmotor0.11.ino

```diff
@@ -135,24 +209,27 @@ void loop() {
   //readSerial();  //Testing cmd from Serial
 
   if(!breathing){
+  	speedMov = speedOut;
     goPos = ZERO; //go to HOME position
-    if(BREATH_BUT.push()) breathing=true; //Run breathing
   }else{
-    if(BREATH_BUT.holdTime() >= brth_hold_time) brth_hold_stat=true;
-    if(brth_hold_stat && BREATH_BUT.push()){
-      brth_hold_stat=false;
-      breathing=false;
-    }
-
     switch(venStat){
       case pushIN:
         goPos = TV2RPB;
         speedMov = speedIn;
+        
+        /*
         if(millis()-timer>=timeIn || abs(goPos-aPos)<=0.04){
           timer = millis();
           venStat = pushOUT;
+        }
+		*/
+
+        if(abs(goPos-aPos)<=0.01) {
+        	timer = millis();
+        	venStat = pushOUT; //reached desire position, move on
         }
         break;
+
```

Comment:
You still need to add a brief "end inspiratory hold" between the inhale and exhale for reading the plateau pressure. See the MIT docs:
https://e-vent.mit.edu/controls/high-level-controls/

I also had a few comments on this part on the previous review:
https://github.com/LIBVproject/LIBV-ventilator/pull/4#discussion_r410823447